### PR TITLE
Guil8553/fix includes

### PR DESCRIPTION
**Pull request by GuillaumeBelz**
# Description

WIP. Fix build for includes.

- viewer build without error on Mac
- widget samples not yet fixed
- each projects not yet fixed individually 

## Type of change

- [x] Bug fix
- [ ] New sample implementation
- [ ] Sample viewer enhancement
- [ ] Other enhancement

**Platforms tested on**:

- [x] Windows
- [ ] Android
- [ ] Linux
- [x] macOS
- [ ] iOS

## Checklist

- [ ] Runs and compiles on all active platforms as a standalone sample
- [ ] Runs and compiles in the sample viewer(s)
- [ ] Branch is up to date with the latest main/v.next
- [ ] All merge conflicts have been resolved
- [ ] Self-review of changes
- [ ] There are no warnings related to changes
- [ ] No unrelated changes have been made to any other code or project files
- [ ] Code is commented with correct formatting (CTRL+i)
- [ ] All variable and method names are camel case
- [ ] There is no leftover commented code
- [ ] Screenshots are correct size and display in description tab (500px by 500px, platform agnostic)
- [ ] If adding a new sample, it is added to the sample viewer
- [ ] Cherry-picked to Main branch (if applicable)

### CODE REVIEW

**Comment by GuillaumeBelz** [comment]
Ready to review.
- sample viewer build on Mac and Window
- widget samples build on Mac

File: ArcGISRuntimeSDKQt_CppSamples/Analysis/DistanceMeasurementAnalysis/DistanceMeasurementAnalysis.cpp [hunk 1]
```diff
@@ -24,11 +24,21 @@
...
 #include "ArcGISSceneLayer.h"
 #include "Point.h"
+#include "MapTypes.h"
+#include "LayerListModel.h"
+#include "Surface.h"
+#include "ElevationSourceListModel.h"
+#include "CoreTypes.h"
+#include "Distance.h"
+#include "TaskWatcher.h"
+#include <QUuid>
```
Comment (JamesMBallard) [inline ArcGISRuntimeSDKQt_CppSamples/Analysis/DistanceMeasurementAnalysis/DistanceMeasurementAnalysis.cpp:41]: Do the git hooks run on the sample code? This is typically placed into an alternate section once the script runs.

File: ArcGISRuntimeSDKQt_CppSamples/Analysis/Geotriggers/Geotriggers.cpp [hunk 2]
```diff
@@ -28,12 +29,20 @@
...
+#include "MapViewTypes.h"
 #include "PortalItem.h"
 #include "ServiceFeatureTable.h"
 #include "SimulatedLocationDataSource.h"
 #include "SimulationParameters.h"
+#include "TaskWatcher.h"
+#include "ArcGISFeature.h"
+#include "AttributeListModel.h"
+#include "Attachment.h"
+#include "AttachmentListModel.h"
```
Comment (JamesMBallard) [inline ArcGISRuntimeSDKQt_CppSamples/Analysis/Geotriggers/Geotriggers.cpp:44]: These are not alphabetical and they should be. @GuillaumeBelz I though our team's git hooks ran against the samples as well. Is that not the case anymore?

File: ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_Samples/SampleManager.cpp [hunk 3]
```diff
@@ -351,7 +349,7 @@ void SampleManager::setApiKey(bool isSupportsApiKey)
 #ifdef SAMPLE_VIEWER_API_KEY
   // If the API key identifier is defined in the respective .pro file it will be used here
   // Otherwise use the API key provided by the user at the top of this file
-  apiKey = QUOTE(SAMPLE_VIEWER_API_KEY);
+//  apiKey = QUOTE(SAMPLE_VIEWER_API_KEY);
```
Comment (JamesMBallard) [inline ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_Samples/SampleManager.cpp:352]: Please review. This does not look intentional.

**Review by JamesMBallard — COMMENTED** [review]
changes mostly are all good, but please run the CodeFormatter script (or the git hooks) on the modified files.

**Comment by GuillaumeBelz** [comment]
Right for Git hook, I forgot to enable them.

**Review by JamesMBallard — APPROVED** [review]

**Comment by JamesMBallard** [comment]
From our team discussion today, our git hooks do not work on the samples right now due to various reasons and it is not in scope to address that. This can move forward as-is.

**Pull request merged**